src/python/WMCore/MicroService/Unified/Common.py

```diff
@@ -259,60 +276,71 @@ def cert():
     pair = getKeyCertFromEnv()
     return pair[1]
 
-def stucktransferUrl():
+def stucktransferUrl(url=None):
     "Return stucktransfer url"
-    default = 'https://cms-stucktransfers.web.cern.ch/cms-stucktransfers'
-    return uConfig.get('stucktransferUrl', default)
+    if not url:
+        url = 'https://cms-stucktransfers.web.cern.ch/cms-stucktransfers'
+    return uConfig.get('stucktransferUrl', url)
 
-def dashboardUrl():
+def dashboardUrl(url=None):
     "Return dashboard url"
-    default = 'http://dashb-ssb.cern.ch/dashboard/request.py'
-    return uConfig.get('dashboardUrl', default)
+    if not url:
+        url = 'http://dashb-ssb.cern.ch/dashboard/request.py'
+    return uConfig.get('dashboardUrl', url)
 
-def monitoringUrl():
+def monitoringUrl(url=None):
     "Return monitoring url"
-    default = 'http://cmsmonitoring.web.cern.ch/cmsmonitoring'
-    return uConfig.get('monitoringUrl', default)
+    if not url:
+        url = 'http://cmsmonitoring.web.cern.ch/cmsmonitoring'
```

Comment:
Don't waste time refactoring these simple get methods. Eventually we should convert all of them to service arguments and that's it.